### ISSUE TITLE
kcribbage: Gameplay Improvements

### ISSUE THREAD

**Issue by keith-packard** [keith-packard/kgames#16]
KCribbage is pretty hard to parse. Ideas:

 1. Show the crib -- 'empty' slots before discard, card backs afterwards
 2. Put the deck in the middle of the board, next to the play area
 3. Add numeric scores
 4. Two track score boards, just like regular cribbage.
 5. Animate card motions
 6. Improve text stuff with separate colors?

**Comment by BartMassey** [comment]
With current fixes, we have

* [x] Show the crib -- 'empty' slots before discard, card backs afterwards
* [x] Put the deck in the middle of the board, next to the play area
* [x] Add numeric scores
* [x] Two track score boards, just like regular cribbage.
* [x] Animate card motions
* [ ] Improve text stuff with separate colors?
* [x] Draggable play.

Would really like to see a two-track scoreboard for the short game; like the long scoreboard for the long game. In addition to animating card motions, would like to see draggable play as an option. Coloring the player/opponent text to match the peg colors would be awesome.

**Comment by BartMassey** [comment]
Let's add a couple of more…

 Would like to have the computer count "cribbage-style": "15-8 and a double-double-run of 3 makes 24", which helps to check the computers scoring and understand the game.
 
Would like a "hint" button like in other kgames, that would hint scoring, discards or play depending on the game phase. Cribbage-style score-counting would be especially helpful.

* [x] Cribbage-style score counting
* [ ] Score hinting
* [ ] Discard hinting
* [ ] Play hinting

**Comment by BartMassey** [comment]
Significant progress has been made!
